var/spack/repos/builtin/packages/eztrace/package.py

```diff
@@ -50,13 +60,15 @@ def install(self, spec, prefix):
             "-DEZTRACE_ENABLE_OPENMP=ON",
             "-DEZTRACE_ENABLE_POSIXIO=ON",
             "-DEZTRACE_ENABLE_PTHREAD=ON",
+            "-DEZTRACE_ENABLE_OMPT=%s" % ("ON" if spec.satisfies("%llvm-openmp-ompt") else "OFF"),
+            "-DEZTRACE_ENABLE_STARPU=%s" % ("ON" if "+starpu" in spec else "OFF"),
+            "-DEZTRACE_ENABLE_CUDA=%s" % ("ON" if "+cuda" in spec else "OFF"),
+            "-DEZTRACE_ENABLE_NETCDF=%s" % ("ON" if "+netcdf" in spec else "OFF"),
+            "-DEZTRACE_ENABLE_PNETCDF=%s" % ("ON" if "+pnetcdf" in spec else "OFF"),
```

Comment:
If this correct also for older versions?

Reply:
For older versions (before 2.1), this is still correct as it will be ignored by cmake

Reply:
Can we add the option only where it exists, so to limit the number of warnings from CMake?

Reply:
I changed the package in order to define these cmake option only if needed. This should prevent warnings.